### PR TITLE
add testpilot to addon desc

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

File: install.rdf [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <em:homepageURL>https://github.com/bwinton/VerticalTabs</em:homepageURL>
     <em:updateURL>https://people.mozilla.com/~bwinton/TabCenter/tab-center.update.rdf</em:updateURL>
     <em:iconURL>resource://tabcenter/skin/icon.png</em:iconURL>
-    <em:description>A new, vertical, arrangement for your tabs.</em:description>
+    <em:description>A new, vertical, arrangement for your tabs. Learn more about this Test Pilot experiment at https://testpilot.firefox.com/</em:description>
```
Comment (ckprice) [inline install.rdf]: Period?

**Comment by ckprice** [comment]
r+ wc (I can't merge things in this place)

File: install.rdf [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <em:homepageURL>https://github.com/bwinton/VerticalTabs</em:homepageURL>
     <em:updateURL>https://people.mozilla.com/~bwinton/TabCenter/tab-center.update.rdf</em:updateURL>
     <em:iconURL>resource://tabcenter/skin/icon.png</em:iconURL>
-    <em:description>A new, vertical, arrangement for your tabs.</em:description>
+    <em:description>A new, vertical, arrangement for your tabs. Learn more about this Test Pilot experiment at https://testpilot.firefox.com/.</em:description>
```
Comment (bwinton) [inline install.rdf]: Two spaces after the first period?

Comment (johngruen) [inline install.rdf]: Is that a thing? thought that was not a thing.

Comment (bwinton) [inline install.rdf]: I always do it, but html doesn't render them, so it's like I'm just wasting bytes…                                                                
🙂

**Pull request merged**